src/main/java/com/bakdata/kafka/KafkaStreamsApplication.java

```diff
@@ -87,6 +90,9 @@ public abstract class KafkaStreamsApplication extends KafkaApplication implement
     @CommandLine.Option(names = "--delete-output", arity = "0..1",
             description = "Delete the output topic during the clean up.")
     private boolean deleteOutputTopic;
+    @CommandLine.Option(names = "--volatile-group-instance-id", arity = "0..1",
+            description = "Whether the group instance id is volatile, i.e., it will change on a Streams shutdown.")
+    private boolean volatileGroupInstanceId;
```

Comment:
The helm parameter should be set automatically based on the deployment. What it exactly does (currently setting this CLI param) is implementation detail. I added the helm parameter in order to be able to deploy old streams-bootstrap apps with the new helm chart because the parameter might not be available and the app would then crash

Reply:
I update the docs